src/main/scala/sigmastate/utxo/CostTable.scala

```diff
@@ -94,6 +102,38 @@ object CostTable {
 
     ("%", "(BigInt, BigInt) => BigInt", 0.0001),
     ("%_per_item", "(BigInt, BigInt) => BigInt", MinimalCost),
+
+    ("Downcast", "(Short) => Byte", MinimalCost),
```

Comment:
@greenhat we can simplify this by using generic type signature. I don't think the cost of Upcast and Downcast depend on concrete types. So we can have single rule like ("Upcast", "T => R").  See examples for array operations.